explainer.md

```diff
@@ -1,121 +1,159 @@
-<h2>Background Synchronization Explained</h2>
+# Background synchronization explained
 
-Modern web applications with heavy client-side logic often need to synchronize data with a server. This need is exacerbated by new offline capabilities via [Service Workers](https://github.com/slightlyoff/ServiceWorker) that enable applications to run while disconnected entirely from the server.
+This is a specification that brings both one-off and periodic synchronization to the web, in the form of [Service Workers](https://github.com/slightlyoff/ServiceWorker) events.
 
-Consider the case of a Twitter application. Who can say when 140 characters of genius will strike? In that moment it's _clearly_ preferable for a Twitter client to provide a "send later" button in cases where sending doesn't initially succeed (e.g., while offline). Similar work-saving is natural in document editing applications and even in consumption, e.g., Kindle applications synchronizing on furthest-read page.
+## One-off synchronization
 
-Periodic content updates are also important to improve performance and responsiveness of a web app. Consider a news site that wants to sync the latest articles at night while it's charging so that they're ready for the reader in the morning for use offline.
+If you write an email, instant message, or simply favourite a tweet, the application needs to communicate that data to the server. If that fails, either due to user connectivity, service availability or anything in-between, the app can store that action in some kind of 'outbox' for retry later.
 
-The web currently lacks any ability to provide this sort of functionality in a user-friendly or power-efficient way. Current approaches require an application (or tab) to be running and rely on slow, battery-intensive pings.
+Unfortunately, on the web, that outbox can only be processed while a page on the site is running. This is particularly problematic on mobile, where browsing contexts are frequently shut down to free memory.
```

Comment:
Perhaps "while a site is displayed in a browsing context". Running doesn't really emphasize the background nature.